src/Button.js

```diff
@@ -9,6 +9,7 @@ import systemPropTypes from '@styled-system/prop-types'
 const variants = variant({
   variants: {
     small: {
+      p: '3px 10px',
```

Comment:
I saw other components, like `TextInput` use separate `px` and `py` props, but this directly replaced the `padding` in `buttonStyles`. Happy to update it if you'd like though.